src/1-theory/4-success-or-failure.adoc

```diff
@@ -370,7 +370,7 @@ Many large companies are able to ignore these standards because they can create
 their own effective internal standards, and have enough people around with
 enough experience to avoid a lot of the common problems around.
 
-Smaller teams are not in this privileged position, should can benefit from
+Smaller teams that are not in this privileged position, can benefit from
 differing to standards written by people who have more context on the task at
```

Comment:
is this correct english? "differing to standards" ?

Reply:
deferring haha